insights/config/specs.py

```diff
@@ -14,6 +14,7 @@
 
 static_specs = {
     "autofs.conf"               : SimpleFileSpec("etc/autofs.conf"),
+    "audit_log"                 : SimpleFileSpec("var/log/audit/audit.log", large_content=True),
     "auditd.conf"               : SimpleFileSpec("etc/audit/auditd.conf"),
```

Comment:
Would you mind to move `autofs.conf` after the two `audit...` specs to keep them in order.  Thanks!

Reply:
Sure.